### PR TITLE
Related pods action

### PR DESCRIPTION
**Pull request by arikalon1**
Bug fix - Node creation time may change on node upgrade - add it to the diff

### CODE REVIEW

File: src/robusta/core/model/nodes.py [hunk 1]
```diff
@@ -27,56 +27,7 @@ def __eq__(self, other):
         if not isinstance(other, NodeInfo):
             return NotImplemented
 
-        ignored_fields = ["deleted", "node_creation_time"]  # node_creation_time never changes
+        ignored_fields = ["deleted"]
```
Comment (Avi-Robusta) [inline src/robusta/core/model/nodes.py:30]: any reason why we stopped ignoring this? i think it should be in nodeInfo but i was wondering if we saw the time changes.

File: src/robusta/core/sinks/robusta/robusta_sink.py [hunk 2]
```diff
@@ -191,7 +192,7 @@ def __to_node_info(cls, node: Node) -> Dict:
 
     @classmethod
     def __from_api_server_node(
-        cls, api_server_node: Node, pod_requests: List[PodRequests]
+        cls, api_server_node: Node, pod_requests: List[PodResources]
```
Comment (Avi-Robusta) [inline src/robusta/core/sinks/robusta/robusta_sink.py:195]: confusing name for pod_requests list and pod_requests function we import

Comment (Avi-Robusta) [reply src/robusta/core/sinks/robusta/robusta_sink.py:195]: if we didnt use both on this sinc i would say its no issue, would just rename to pod_requests_list

File: playbooks/robusta_playbooks/k8s_resource_enrichments.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        return
+
+    if resource.kind == "Pod":
+        pods = [resource]
+    else:
+        selector = build_selector_query(resource.spec.selector)
+        pods = PodList.listNamespacedPod(namespace=resource.metadata.namespace, label_selector=selector).obj.items
+
+    rows = []
+    for pod in pods:
```
Comment (Avi-Robusta) [inline playbooks/robusta_playbooks/k8s_resource_enrichments.py:32]: looks good, i personally would wrap everything in a function `generate_related_pod_row` but its fine like this

File: src/robusta/core/playbooks/playbooks_event_handler_impl.py [hunk 4]
```diff
@@ -79,6 +79,10 @@ def run_actions(
```
Comment (Avi-Robusta) [inline src/robusta/core/playbooks/playbooks_event_handler_impl.py:79]: unrelated to your code i had a question about line 39
`if execution_event:  # might not exist for unsupported k8s types`
which ones do we not support?

File: src/robusta/integrations/slack/sender.py [hunk 5]
```diff
@@ -189,6 +189,15 @@ def __send_blocks_to_slack(
         )
         other_blocks = [b for b in report_blocks if not isinstance(b, FileBlock)]
 
+        # wide tables aren't displayed properly on slack. looks better in a text file
+        table_blocks = [b for b in other_blocks if isinstance(b, TableBlock)]
+        for table_block in table_blocks:
+            if len(table_block.headers) > 4:
```
Comment (Avi-Robusta) [inline src/robusta/integrations/slack/sender.py:195]: I would make the 4 something like MAX_HEADERS_AMNT_FOR_SLACK_TABLE=4 at the top of the file to make it easier to find and change in the future

**Comment by Avi-Robusta** [comment]
Looks good, nothing critical to change

File: src/robusta/core/model/nodes.py (hunk 1 above)
Comment (arikalon1) [reply src/robusta/core/model/nodes.py:30]: Yeah, the time changes when you upgrade the node.
I wrote it in the comment

File: src/robusta/integrations/slack/sender.py (hunk 5 above)
Comment (arikalon1) [reply src/robusta/integrations/slack/sender.py:195]: fixed

File: src/robusta/core/playbooks/playbooks_event_handler_impl.py (hunk 4 above)
Comment (arikalon1) [reply src/robusta/core/playbooks/playbooks_event_handler_impl.py:79]: We don't support CronJobs, Secrets, Ingresses
k8s has many resource types. You can run `kubectl api-resources` to see
In addition, we don't support CRDs yet
The list of our supported types is in `generate_kubernetes_code.py`

File: playbooks/robusta_playbooks/k8s_resource_enrichments.py (hunk 3 above)
Comment (arikalon1) [reply playbooks/robusta_playbooks/k8s_resource_enrichments.py:32]: fixed

File: src/robusta/core/sinks/robusta/robusta_sink.py (hunk 2 above)
Comment (arikalon1) [reply src/robusta/core/sinks/robusta/robusta_sink.py:195]: fixed

**Pull request merged**